pdf/creator/creator_test.go

```diff
@@ -1042,7 +1707,7 @@ func TestTableInSubchapter(t *testing.T) {
 	cell := issuerTable.NewCell()
 	cell.SetContent(p)
 	cell.SetBackgroundColor(ColorBlack)
-	cell.SetBorder(CellBorderStyleBox, 1.0)
+	cell.SetBorder(CellBorderStyleSingle, 1.0, CellBorderSideBox)
```

Comment:
I think I like your idea better with having
cell.SetBorder(CellBorderStyle, width)
and having functions
cell.SetBorderLeft(CellBorderStyle, width)
cell.SetBorderRightCellBorderStyle, width)
cell.SetBorderTopCellBorderStyle, width)
cell.SetBorderBottomCellBorderStyle, width)
to set the individual borders.
Also keeps the API compatibility so we can merge into master.